e2e/index.ts

```diff
@@ -0,0 +1,18 @@
+import { file } from "bun";
+
+const PORT = process.env.SERVER_PORT || 8080;
+
+Bun.serve({
+  fetch(req) {
+    const url = new URL(req.url);
+    const pathname = url.pathname === "/" ? "/index.html" : url.pathname;
+    const filePath = `./dist${pathname}`;
```

Comment:
Shouldn't this be:
```suggestion
    const filePath = `./${pathname}`;
```

Reply:
As we are serving the `dist`, if we remove it from the template string, it would consider we are trying to serve the root folder

Reply:
Oh, I missed that you removed the file router. Never mind.